docs/unified-test-documentation/dasharo-compatibility/325-custom-boot-order.md

```diff
@@ -0,0 +1,56 @@
+# Dasharo Compatibility: Custom Boot Order (SSD, USB)
+
+## Test cases
+
+### Common
+
+**Test setup**
+
+1. Proceed with the
+    [Generic test setup: firmware](../../generic-test-setup/#firmware).
+
+### CBO001.001 Custom boot order
+
+**Test description**
+
+This test aims to verify that the DUT boot from the suitable source, with the
+posibility to boot from other sources.
+
+**Test configuration data**
+
+1. `FIRMWARE` = coreboot
+
+**Test setup**
+
+1. Proceed with the [Common](#common) section.
+
+**Test steps**
+
+1. Power on the DUT
+1. Press `BOOT_MENU_KEY` key to display boot menu.
+1. Compare the listed devices with the desired boot order.
```

Comment:
Currently, this test suite is dedicated only to two Dasharo platforms: Protectli VP4620 and VP2410. Keep in mind that in the future, if this test is also introduced for other platforms, we need to add Custom Boot Order files dedicated to any platform.